stripper/ze_l0v0l_v1_4_csgo2.cfg

```diff
@@ -28,4 +28,59 @@ modify:
 	{
 		"item_assaultsuit" "1"
 	}
-}
\ No newline at end of file
+}
+
+;fix level 4 nuke from killing every single player after the Nuke King boss is dead and you get teleported
+modify:
+{
+	match:
+	{
+		"targetname" "lvl4_win"
+	}
+	delete:
+	{
+		"OnStartTouch" "Map_BrushRunScriptCodeKillAll();61"
+	}
+	insert:
+	{
+		"OnStartTouch" "!activatorAddOutputhealth 99999999990-1"
+		"OnStartTouch" "MapBrush,RunScriptCode,KillAllT();11"
```

Comment:
You're missing the last number on this output (only once: 1 or -1), there's also no separator between the argument (KillAllT();) and the delay (11)